### DISCUSSION TITLE
Can't add a thumbnail to an embed

### DISCUSSION THREAD

**Discussion by hyperion1008** [Rapptz/discord.py#6108] [Q&A] [answered]
Hi, I'm trying to add a thumbnail to an embed with `embed.set_thumbnail(url="attachment://Image.png")`, but when i send the embed the image isn't there, any solutions?

**Comment by Harmon758** [comment, marked as answer]
Are you sending the image file along with the embed?
e.g. like in https://discordpy.readthedocs.io/en/latest/faq.html#how-do-i-use-a-local-image-file-for-an-embed-image

**Reply by hyperion1008** [reply]
Yes, I am now, and for some reason it sends it seperately, any tip for that?

**Reply by Harmon758** [reply]
You'll need to provide the relevant code. Otherwise it's impossible to tell what the issue is.

**Reply by hyperion1008** [reply]
Fixed it, don't worry
